### DISCUSSION TITLE
safari crashes, webgl shader compilation errors in console and icons being blank in safari

### DISCUSSION THREAD

**Discussion by blairmacintyre** [mozilla/hubs#4338] [Hubs Cloud Questions]
On Safari on the newest macOS (11.4), going to my custom hubs cloud results in these errors in the console:

<img width="1722" alt="image" src="https://user-images.githubusercontent.com/1348861/122446679-ee865580-cf70-11eb-88f3-d6b1dfaba151.png">

When the page loads, bringing up the menu on an object (using space bar), shows a bunch of menu item images being blank.

This does not happen on hubs.mozilla.com

In addition, many times I go to a hubs room (HMC or my hubs cloud) and the pages fails to load, or reloads after crashing.  This doesn't happen on other browsers.

**Comment by pattersonbl2** [comment]
Are you still having issues with this ?
